tests/src/integration/common/wskdeploy.go

```diff
@@ -174,6 +179,10 @@ func (Wskdeploy *Wskdeploy) HeadlessManagedDeployment(manifestPath string, deplo
 	return Wskdeploy.RunCommand("sync", "-m", manifestPath, "-d", deploymentPath, "--projectname", name)
 }
 
+func (wskdeploy *Wskdeploy) ExportProject(projectName string, targetManifestPath string) (string, error) {
+	return wskdeploy.RunCommand("export", "-m", targetManifestPath, "-p", projectName)
```

Comment:
Please change the overall command signature from `wskdeploy export -p <project-name>` to `wskdeploy export --projectname <project name>` to be consistent with other `wskdeploy` commands.